packages/react-native/src/notifier.js

```diff
@@ -1,6 +1,14 @@
 const NativeModules = require('react-native').NativeModules
 const NativeClient = NativeModules.BugsnagReactNative
 
+const REMOTE_DEBUGGING_WARNING = `Bugsnag cannot initialise synchronously when running in the remote debugger.
+
+Error reporting is still supported, but synchronous calls afted Bugsnag.start() will no-op. This means Bugsnag.leaveBreadcrumb(), Bugsnag.setUser() and all other methods will only begin to work after a short delay.
+
+This only affects the remote debugger. Execution of JS in the normal way (on the device) is not affected.`
+
+const isDebuggingRemotely = !global.nativeCallSyncHook
```

Comment:
Documentation for this is hard to find. I found https://github.com/facebook/react-native/commit/417e191a1cfd6a049d1d4b0a511f87aa7f176082 which partially helps my understanding.

Reply:
Yeah that's the only place I could see it. I'll add a comment pointing to there.